samza-kv/src/main/scala/org/apache/samza/storage/kv/KeyValueStorageEngineMetrics.scala

```diff
@@ -28,27 +28,32 @@ class KeyValueStorageEngineMetrics(
   val registry: MetricsRegistry = new MetricsRegistryMap) extends MetricsHelper {
 
   val gets = newCounter("gets")
-  val ranges = newCounter("ranges")
-  val alls = newCounter("alls")
+  val getAlls = newCounter("getAlls")
```

Comment:
Minor: Should this be "get-alls" for consistency?

Reply:
Good catch, thanks.